addons/hr_timesheet/security/hr_timesheet_security.xml

```diff
@@ -122,5 +122,12 @@
             <field name="domain_force">[(1, '=', 1)]</field>
             <field name="groups" eval="[(4, ref('group_timesheet_manager')), (4, ref('project.group_project_manager'))]"/>
         </record>
+
+        <record id="analytic.analytic_line_comp_rule" model="ir.rule">
+            <field name="name">Analytic line multi company rule bis</field>
```

Comment:
"... bis"? I think it could have a name that explains a little more what it is :see_no_evil: